ci.mk

```diff
@@ -0,0 +1,78 @@
+CI_BUILD=./smithy-rs/tools/ci-build
```

Comment:
add a comment about what this makefile does / what the targets do (if any should be used by a user)